wear/src/main/java/io/homeassistant/companion/android/home/views/MainView.kt

```diff
@@ -89,19 +89,20 @@ fun MainView(
                         items(favoriteEntityIds.size) { index ->
                             val favoriteEntityID = favoriteEntityIds[index].split(",")[0]
                             if (mainViewModel.entities.isEmpty()) {
-                                // Use a normal chip when we don't have the state of the entity
+                                // when we don't have the state of the entity, create a Chip from cache as we don't have the state yet
+                                val cached = mainViewModel.getCachedEntity((favoriteEntityID))
```

Comment:
No need to use double parantheses

```suggestion
                                val cached = mainViewModel.getCachedEntity(favoriteEntityID)
```